modules/configuring-access-control-and-sending-invitations.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * configuration/inviting-users-to-rhacs.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-access-control-and-sending-invitations_{context}"]
+= Configuring access control and sending invitations
+
+.Procedure
+
+. To invite a user, use one of the following methods:
+* In the {product-title-short} portal, navigate to *Platform Configuration -> Access Control -> Auth providers* tab, and then click *Invite users*.
+* Navigate to your profile icon, and then click *Invite users*.
+
+. In the *Invite users* dialog box, provide the following information:
+* *Emails to invite*: Enter one or more email addresses of the users you want to invite. Ensure that they are valid email addresses associated with the intended recipients.
+* *Provider*: From the drop-down list, select a provider you want to use for each invited user.
++
+[IMPORTANT]
+====
+* If you have only one authentication provider available, it is selected by default.
+* If multiple authentication providers are available and at least one of them is `Red Hat SSO` or `Default Internal SSO`, that provider is selected by default.
+* If multiple authentication providers are available, but none of them is `Red Hat SSO` or `Default Internal SSO`, you are prompted to select one manually.
+* If you have not yet set up authentication providers, you will receive a warning message; the form will be disabled and you will receive a link that takes you to the *Access Control* section where you can configure an authentication provider.
+====
+* *Role*: From the drop-down list, select the role to assign to each invited user.
+
+. Click *Invite users*.
+. When you receive confirmation that the users have been created with the selected role, copy the one or more email addresses and the message into an email that you create in your own email client, and send it to the users.
```

Comment:
It's not clear to me where/how they receive confirmation. Is it right in the UI, or somewhere else? Does it take a significant amount of time to wait, or is it instantaneous?

Also, where do they get "the message" they are supposed to copy? Is that displayed in the UI after they have clicked "invite users"?

* If they have to wait a bit, then I'd break this up into two steps. first of waiting, and then second about copying/sending
* If it's relatively instantaneous, then I'd remove that first part of the sentence and replace it with something more basic like telling them where to get the message to copy and send.

Reply:
> It's not clear to me where/how they receive confirmation. Is it right in the UI, or somewhere else? Does it take a significant amount of time to wait, or is it instantaneous?
> 
> Also, where do they get "the message" they are supposed to copy? Is that displayed in the UI after they have clicked "invite users"?
It is displayed on the confirmation dialog box at the UI. 
> 
> * If they have to wait a bit, then I'd break this up into two steps. first of waiting, and then second about copying/sending
> * If it's relatively instantaneous, then I'd remove that first part of the sentence and replace it with something more basic like telling them where to get the message to copy and send.

@vjwilson can we re-write the steps to: 
. Click *Invite users*.
. On the confirmation dialog box, you see a message that the users have been created with the selected role.
. Copy the one or more email addresses and the message into an email that you create in your own email client, and send it to the users.
. Click *Done*.

Also, like Andrea mentioned if the user clicked on "Done" before copying the email addresses, what would happen?

Reply:
1. What you suggest for the re-writing the steps is fine by me.
2. If the user closes the dialog before copying the information presented, they can't get back to it. The users have still been added to the system, and they can still send an email manually--they just have to enter the emails and type the message themselves by hand.

This feature is a stop-gap measure to make it easier for cloud and trial customers to start using the system, but we are not trying to build a complete user-management system.